features/user_creates_account.feature

```diff
@@ -0,0 +1,14 @@
+Feature: User creates an account
+  As a user
+  In order to use the site
+  I have to sign up and create an account
+
+  Scenario: User can sign up
+    Given I visit the site
+    When I click "Sign up"
+    # And I fill in "Name" with "Maran"
```

Comment:
Please remove commented out code

Reply:
We want to add name field to the sign up using devise.

Reply:
Is it part of this PR? Adding custom attributes is a bit tricky and should be extracted to a separate feature/story imo.